olive/engine/engine.py

```diff
@@ -621,6 +627,21 @@ def get_pareto_frontier_footprints(
 
         return pf_footprints
 
+    def dump_run_history(self, run_history, output_path: str = None):
+        if not run_history:
+            logger.info("No run history to dump!")
```

Comment:
return here? otherwise it will fail in the next steps

Reply:
good caceh. fixed.